src/embassy/time_driver_tim.rs

```diff
@@ -191,56 +179,54 @@ impl RtcDriver {
             Ok(n) => n,
         };
 
-        r.psc().write_value(psc);
-        r.atrlr().write_value(u16::MAX);
+        r.psc().write_value(psc);//prescaler
+        r.atrlr().write_value(u16::MAX);//auto-reload register
 
         // Set URS, generate update and clear URS
-        r.ctlr1().modify(|w| w.set_urs(vals::Urs::COUNTERONLY));
-        r.swevgr().write(|w| w.set_ug(true));
-        r.ctlr1().modify(|w| w.set_urs(vals::Urs::ANYEVENT));
+        r.ctlr1().modify(|w| w.set_urs(vals::Urs::COUNTERONLY));//Update request source
+        r.swevgr().write(|w| w.set_ug(true));//Update generation
+        r.ctlr1().modify(|w| w.set_urs(vals::Urs::ANYEVENT));//Update request source
 
         // Mid-way point. CC1
-        r.chcvr(0).write_value(0x8000);
+        r.chcvr(0).write_value(0x8000);//capture/compare register 1
 
         // Enable overflow and half-overflow interrupts
         r.dmaintenr().write(|w| {
-            w.set_uie(true);
-            w.set_ccie(0, true);
+            w.set_uie(true);//Update interrupt enable
+            w.set_ccie(0, true);//Capture/Compare 1 interrupt enable
         });
 
         <T as GeneralInstance16bit>::CaptureCompareInterrupt::unpend();
         unsafe { <T as GeneralInstance16bit>::CaptureCompareInterrupt::enable() };
 
-        r.ctlr1().modify(|w| w.set_cen(true));
+        r.ctlr1().modify(|w| w.set_cen(true));//Counter enable
```

Comment:
Please format this comment like `//<space> text`

https://github.com/embassy-rs/embassy/blob/main/embassy-stm32/src/time_driver.rs
Looks like this mostly copy pasted from the STM32 variant so LGTM function wise.